Container Services/getting-started-with-kubernetes.md

```diff
@@ -0,0 +1,210 @@
+     -m= (--minion_count=)         number of kubernetes minion nodes
+     -mem= (--vm_memory=)          number of GB ram for each minion
+     -cpu= (--vm_cpu=)             number of virtual cps for each minion node
+     -phyid= (--server_conf_id=)   physical server configuration id, one of
+                                      physical_server_20_core_conf_id
+                                      physical_server_12_core_conf_id
+                                      physical_server_4_core_conf_id (default)
+     -etcd_separate_cluster=yes    create a separate cluster of three etcd nodes,
+                                   otherwise run etcd on the master node
+```
+#### Script Examples
+
+Create a cluster with name of k8s_1, 1 master node and 3 worker minions (on physical machines), in VA1
+
+```
+ bash kube-up.sh --clc_cluster_name=k8s_1 --minion_type=bareMetal --minion_count=3 --datacenter=VA1
+```
+Create a cluster with name of k8s_2, an ha etcd cluster on 3 VMs and 6 worker minions (on VMs), in VA1
+
+```
+ bash kube-up.sh --clc_cluster_name=k8s_2 --minion_type=standard --minion_count=6 --datacenter=VA1 --etcd_separate_cluster=yes
+```
+Create a cluster with name of k8s_3, 1 master node, and 10 worker minions (on VMs) with higher mem/cpu, in UC1:
+
+
+```
+  bash kube-up.sh --clc_cluster_name=k8s_3 --minion_type=standard --minion_count=10 --datacenter=VA1 -mem=6 -cpu=4
+```
+
+## Cluster Deletion
+
+To delete a cluster, log into the CenturyLink Cloud control portal and delete the parent server group that contains the Kubernetes Cluster. We hope to add a scripted option to do this soon. 
+
+
+## What Kubernetes features do not work on CenturyLink Cloud
+
+- At this time, there is no support services of the type 'loadbalancer'. We are actively working on this and hope to publish the changes soon. 
+- At this time, there is no support for persistent storage volumes provided by CenturyLink Cloud. However, customers can bring their pwn persistent storage offering.
+
+
+
+## Postrequisite
+
+#### Kubernetes  Access
+You will need the kubectl client program to manage your kubernetes cluster. You can access this by one of two ways:
+
+- The master server has this installed. SSH to the master server and run kubectl from there. 
+- Use your local machine (or some remote machine to the cluster)
+
+#### Network access to the cluster
+- Network access from a remote box to the master can be achieved by using the VPN access that you needed in the previous step or by adding a public IP address to the master server. 
+
+
+## Kubernetes Usage
+
+#### How to access the kubernetes UI 
+
+Goto the following url in a web browser that has access to the master server:
+
+http://(IP-ADDRESS-OF-MASTER-SERVER):8080/api/v1/proxy/namespaces/kube-system/services/kube-ui
+
+#### How to use the kubectl client
+
+To get your Kubernetes cluster's information:
+
+``` kubectl -s http://(IP-ADDRESS-OF-MASTER-SERVER):8080 cluster-info ```
```

Comment:
either move the text to a new lines or change the triple tic to single.

Do the same for all below.